src/commands/ssh.ts

```diff
@@ -48,19 +39,11 @@ export const sshCommand = (yargs: yargs.Argv) =>
           string: true,
           default: [] as string[],
         })
-        .check((argv: yargs.ArgumentsCamelCase<SshCommandArgs>) => {
-          if (argv.L == null) return true;
-
-          return (
-            argv.L.match(LOCAL_PORT_FORWARD_PATTERN) ||
-            "Local port forward should be in the format `local_port:remote_port`"
-          );
-        })
         .option("L", {
           type: "string",
+          // Copied from `man ssh`
           describe:
-            // the order of the sockets in the address matches the ssh man page
-            "Forward a local port to the remote host; `local_socket:remote_socket`",
+            "Specifies that connections to the given TCP port or Unix socket on the local (client) host are to be forwarded to the given host and port, or Unix socket, on the remote side. This works by allocating a socket to listen to either a TCP port on the local side, optionally bound to the specified bind_address, or to a Unix socket.  Whenever a connection is made to the local port or socket, the connection is forwarded over the secure channel, and a connection is made to either host port hostport, or the Unix socket remote_socket, from the remote machine.",
```

Comment:
I think we want to keep these descriptions as short as possible because it looks weird when you type `p0 ssh --help`, since this message is so large does it take over your terminal?

Reply:
Hm, this is the first paragraph of the ssh man page... but it's true that it's still very long. I only kept the first sentence.